AttributeError: 'Search' object has no attribute 'last_token'

```
[2017-05-11 15:13:27 +0200] [1823] [ERROR] Error handling request /search/?q=%3B&lat=47.18971246448421&lon=2.1093750000000004&limit=5
Traceback (most recent call last):
  File "/srv/addok/venv/lib/python3.4/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/srv/addok/venv/lib/python3.4/site-packages/gunicorn/workers/sync.py", line 176, in handle_request
    respiter = self.wsgi(environ, resp.start_response)
  File "/srv/addok/venv/lib/python3.4/site-packages/falcon/api.py", line 209, in __call__
    responder(req, resp, **params)
  File "/srv/addok/venv/lib/python3.4/site-packages/addok/http/base.py", line 134, in on_get
    lat=lat, lon=lon, **filters)
  File "/srv/addok/venv/lib/python3.4/site-packages/addok/core.py", line 327, in search
    return helper(query, lat=lat, lon=lon, **filters)
  File "/srv/addok/venv/lib/python3.4/site-packages/addok/core.py", line 167, in __call__
    if collector(self):
  File "/srv/addok/venv/lib/python3.4/site-packages/addok/autocomplete.py", line 46, in only_commons_but_geohash_try_autocomplete_collector
    autocomplete(helper, helper.tokens, use_geohash=True)
  File "/srv/addok/venv/lib/python3.4/site-packages/addok/autocomplete.py", line 81, in autocomplete
    helper.debug('Autocompleting %s', helper.last_token)
AttributeError: 'Search' object has no attribute 'last_token'
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -14,7 +14,7 @@ install:
 - python setup.py develop
 - pip install -r requirements-dev.txt
 - pip install python-coveralls
-script: py.test
+script: make testcoverage
 after_success:
 - coveralls
 notifications:

--- a/addok/autocomplete.py
+++ b/addok/autocomplete.py
@@ -42,12 +42,12 @@ class EdgeNgramIndexer:
 
 
 def only_commons_but_geohash_try_autocomplete_collector(helper):
-    if helper.geohash_key and len(helper.tokens) == len(helper.common):
+    if helper.geohash_key and helper.only_commons:
         autocomplete(helper, helper.tokens, use_geohash=True)
 
 
 def only_commons_try_autocomplete_collector(helper):
-    if len(helper.tokens) == len(helper.common):
+    if helper.only_commons:
         autocomplete(helper, helper.tokens, skip_commons=True)
         if not helper.bucket_empty:
             helper.debug('Only common terms. Return.')

--- a/addok/core.py
+++ b/addok/core.py
@@ -266,6 +266,10 @@ class Search(BaseHelper):
     def pass_should_match_threshold(self):
         return len(self.matched_keys) >= self.should_match_threshold
 
+    @property
+    def only_commons(self):
+        return self.tokens and len(self.tokens) == len(self.common)
+
 
 class Reverse(BaseHelper):
 

--- a/addok/helpers/collectors.py
+++ b/addok/helpers/collectors.py
@@ -17,7 +17,7 @@ def no_available_tokens_abort(helper):
 
 
 def only_commons(helper):
-    if len(helper.tokens) == len(helper.common):
+    if helper.only_commons:
         # Only common terms, shortcut to search
         keys = [t.db_key for t in helper.tokens]
         if helper.geohash_key:
@@ -78,7 +78,7 @@ def bucket_with_meaningful(helper):
 
 
 def reduce_with_other_commons(helper):
-    if len(helper.tokens) == len(helper.common):
+    if helper.only_commons:
         return
     for token in helper.common:  # Already ordered by frequency asc.
         if token not in helper.meaningful and helper.bucket_overflow:

